rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -330,7 +330,7 @@ def _label_id_index_mapping(
         """Create label_id dictionary."""
 
         distinct_label_ids = {
-            example.get(attribute) for example in training_data.intent_examples
+            example.get(attribute) for example in training_data.training_examples
```

Comment:
@evgeniiaraz Whether you take `intent_examples` or `training_examples` shouldn't make a difference, right?